src/Tests/Microsoft.NET.ToolPack.Tests/GivenThatWeWantToPackAToolProjectWithP2PReference.cs

```diff
@@ -63,5 +63,31 @@ public void It_contains_dependencies_dll()
                 }
             }
         }
+
+        [Fact]
+        public void It_remove_other_package_dependency()
```

Comment:
It_removes_other_package_dependencies

Reply:
Given your explanation, how about naming this It_does_not_add_p2p_references_as_package_references_to_nuspec?

Reply:
I like that second suggestion better too.

Reply:
Renamed as the second one